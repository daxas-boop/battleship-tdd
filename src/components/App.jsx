import React from 'react';
import Game from './Game';
import Header from './Header';

function App() {
  return (
    <>
      <Header />
      <Game />
    </>
  );
}

export default App;
