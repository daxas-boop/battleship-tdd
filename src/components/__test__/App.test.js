import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

test('renders correctly', () => {
    const component = renderer.create(<App></App>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})