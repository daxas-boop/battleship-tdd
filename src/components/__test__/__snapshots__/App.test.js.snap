// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="css-1husv38"
/>
`;
